extensions/scss/package.json

```diff
@@ -1,12 +1,16 @@
 {
     "name": "scss",
     "displayName": "%displayName%",
-	"description": "%description%",
+	  "description": "%description%",
     "version": "1.0.0",
     "publisher": "vscode",
     "engines": { "vscode": "*" },
     "scripts": {
-        "update-grammar": "node ../../build/npm/update-grammar.js atom/language-sass grammars/scss.cson ./syntaxes/scss.tmLanguage.json"
+        "update-grammar": "npm run update-scss && npm run update-sassdoc",
```

Comment:
update-grammar.js can already update multiple grammars from the same file. See https://github.com/Microsoft/vscode/blob/master/extensions/perl/package.json#L9 for the syntax.

Reply:
Thanks for the tip — updated the script syntax 👍